packages/forms-web-app/src/controllers/examination/name.js

```diff
@@ -34,10 +43,13 @@ const postName = async (req, res) => {
 	const { body = {}, session } = req;
 	const { errors = {}, errorSummary = [] } = body;
 	const examinationSession = session?.[examinationSessionStorage.name];
+	const sessionCurrentView = req.session?.currentView;
 
-	if (!examinationSession || !req.currentView?.id) return res.status(404).render('error/not-found');
+	if (!examinationSession || !sessionCurrentView?.id)
+		return res.status(404).render('error/not-found');
 
-	const { id, view } = req.currentView;
+	const { id, view } = sessionCurrentView;
+	examinationSessionStorage.property.name = req.body[id] ?? '';
```

Comment:
Is this correct? I think this should be `examinationSession[examinationSessionStorage.property.name] = req.body[id] ?? '';`